streams.yml

```diff
@@ -19,6 +19,42 @@
 #
 ####################################################################################################
 
+golang-1.21:
+  image: openshift/golang-builder:v1.21.3-202311221709.el8.gf79bf9c 
+  mirror: true
+  transform: rhel-8/golang
+  # Leave this upstream information in place even if mirror/transform is disabled. Test Platform read the data.
+  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-{MAJOR}.{MINOR}.art
+  upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-{MAJOR}.{MINOR}
+  # To ensure consistency in our builds and allow CI to use the same build, mirror the upstream builder image from ocp/builder to ocp-private/builder-priv.
+  upstream_image_mirror:
+  - registry.ci.openshift.org/ocp-private/builder-priv:rhel-8-golang-1.21-openshift-{MAJOR}.{MINOR}
```

Comment:
`{MAJOR}.{MINOR}` suggests that this is really available and supported for this version. Which may not be the case.

Reply:
Should I remove the major.minor ? I think we'd be okay without it ?